src/apps/auth-login/Login.tsx

```diff
@@ -34,21 +34,42 @@ export const Login = (props: {
 
   const [currentMethod, setCurrentMethod] = useState<LoginMethod | undefined>();
 
+  const url = new URLSearchParams(window.location.search);
+  let redirectUrl = url.get('redirect-to');
+  if (redirectUrl) {
+    localStorage.setItem('redirect-to', redirectUrl);
+  } else {
+    redirectUrl = localStorage.getItem('redirect-to');
+    if (redirectUrl && redirectUrl.length === 0) {
```

Comment:
I don't think this is strictly necessary. A string of length 0 is already 'falsy'.